packages/plugin-core/src/utils/StartupUtils.ts

```diff
@@ -544,4 +545,28 @@ export class StartupUtils {
 
     MetadataService.instance().v100ReleaseMessageShown = true;
   }
+
+  /**
+   * this method pings the localhost and checks if it is available. Incase local is blocked off,
+   * displays a toaster with a link to troubleshooting docs
+   */
+  static async showWhitelistingLocalhostDocsIfNecessary() {
+    const { failed } = await execa("ping", ["127.0.0.1"]);
```

Comment:
can we create a test for this? also - is ping available for powershell?

Reply:
A test that pings local host? Is that going to be a reliable test?

Reply:
Looks like powershell has an applet called [Test-Connection](https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.management/test-connection?view=powershell-7.2)

Reply:
as long as we cover all platforms (make it work in powershell), I think we can just mock the response coming from the ping for testing. No use in testing the external dependency itself.

Reply:
powershell does allow me to ping localhost
![image](https://user-images.githubusercontent.com/84971366/192693620-33f5b6b2-715a-4ad8-b3ce-407f62d1e394.png)